path_static_roles.go

```diff
@@ -233,6 +234,9 @@ func (b *backend) pathStaticRoleCreateUpdate(ctx context.Context, req *logical.R
 		}
```

Comment:
If we get back an error here should we attempt to delete the WAL that was created? It'll be best-effort but IMO worth trying.

Reply:
Yeah that seems like a good idea. In a similar vein, I've held off adding deletion of WALs on role deletion for now just to keep the patch smaller because it's not critical to the core bug, but do you think it's worth adding that too?